res/job_queue/queue.py

```diff
@@ -490,6 +505,7 @@ async def execute_queue_async(  # pylint: disable=too-many-arguments
         evaluators: Callable[..., Any],
         cert: Optional[Union[str, bytes]] = None,
         token: Optional[str] = None,
+        output_bus: Optional["asyncio.Queue[CloudEvent]"] = None,
```

Comment:
Just wondering here if we should split this function into output_bus version and the old websockets one?